crates/viewer/re_viewer_context/src/open_url.rs

```diff
@@ -436,12 +449,17 @@ impl ViewerOpenUrl {
     pub fn open(
         self,
         egui_ctx: &egui::Context,
-        follow_if_http: bool,
-        select_redap_source_when_loaded: bool,
+        options: &OpenUrlOptions,
         command_sender: &CommandSender,
     ) {
         re_log::debug!("Opening URL: {:?}", &self);
 
+        if options.show_loader
+            && let Ok(url) = self.sharable_url(None)
+        {
+            command_sender.send_system(SystemCommand::ChangeDisplayMode(DisplayMode::Loading(url)));
```

Comment:
interesting that nowhere in the `match` below we change the display mode directly, huh.
I think it would be best to put this last just in case, but I'm not 100% sure: if any of the other commands _also_ changes the display mode directly (not by enqueuing another command but by direct manipulation), then this would ofc break down again 😵‍💫 
Maybe best to leave it as is (seems to work fine!) and leave a comment instead that anything below is not expected to change display mode?